dso-l1/src/main/java/com/tc/object/DistributedObjectClient.java

```diff
@@ -133,10 +134,7 @@
  */
 public class DistributedObjectClient implements TCClient {
 
-  protected static final TCLogger                    DSO_LOGGER                          = CustomerLogging
-                                                                                             .getDSOGenericLogger();
-  private static final TCLogger                      CONSOLE_LOGGER                      = CustomerLogging
-                                                                                             .getConsoleLogger();
+  protected static final Logger DSO_LOGGER = LoggerFactory.getLogger(DistributedObjectClient.class);
```

Comment:
Again another client side class that had logging to both console and the user's target. No more console logging here either.